apps/notifications/constants.py

```diff
@@ -0,0 +1,69 @@
+from django.conf import settings
+from django.db.models import TextChoices
+
+STATIC_PATH = f"{settings.BASE_URL}{settings.STATIC_URL}"
+
+# Small badge used as the app-icon. Shown at the top bar of an android phone.
+# variable: badge
+NOTIFICATION_BADGE_URL = f"{STATIC_PATH}img/notification-badge.png"
+
+# Used as the app icon for notifications
+# variable: icon
+NOTIFICATION_ICON_URL = f"{STATIC_PATH}img/notification-icon-192.png"
+
+# The vibration rhythm to use for the notification
+# variable: vibrate
+NOTIFICATION_VIBRATION_PATTERN = [
+    500,
+    110,
+    500,
+    110,
+    450,
+    110,
+    200,
+    110,
+    170,
+    40,
+    450,
+    110,
+    200,
+    110,
+    170,
+    40,
+    500,
+]
```

Comment:
I can't recall where I got this exact example, and I don't remember if it is any long or not.
I don't think it is long, and that it just plays trough the pattern fast.
Hard to test these things without a production setup.

Reply:
Hmm, may be a bit long. But no problem in trying it out I guess.
Would be a lot easier to try out and change when it hits production and we can change the value live.
It is really hard to work with this on mobile devices, since you need to connect your phone to both the front and back-end of the application at the same time, with authentication.